### PR TITLE
Add "Settings" section to README

### PR DESCRIPTION
**Pull request by sobolevn**
Feedback is welcome :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,12 @@ class MyModel(models.Model):
 
 * `django_stubs_ext.db.router.TypedDatabaseRouter` can be used as base when implementing custom database routers.
 
+## Settings
+
+The list of settings we support in `[tool.django-stubs]` for `toml` or `[mypy.plugins.django-stubs]` for `ini`:
```
Comment (intgr) [inline README.md]:
```suggestion
The list of settings we support in `[tool.django-stubs]` for `pyproject.toml` or `[mypy.plugins.django-stubs]` for `mypy.ini`:
```

File: README.md [hunk 2]
```diff
@@ -80,6 +80,12 @@ class MyModel(models.Model):
...
+- `django_settings_module` string, which is the same as [`DJANGO_SETTINGS_MODULE` env var](https://docs.djangoproject.com/en/4.2/topics/settings/#designating-the-settings)
```
Comment (intgr) [inline README.md]:
```suggestion
- `django_settings_module` string, which is the same as [`DJANGO_SETTINGS_MODULE` env var](https://docs.djangoproject.com/en/stable/topics/settings/#designating-the-settings)
```

File: README.md [hunk 3]
```diff
@@ -80,6 +80,12 @@ class MyModel(models.Model):
 
 * `django_stubs_ext.db.router.TypedDatabaseRouter` can be used as base when implementing custom database routers.
 
+## Settings
+
+The list of settings we support in `[tool.django-stubs]` for `pyproject.toml` or `[mypy.plugins.django-stubs]` for `mypy.ini`:
+- `django_settings_module` string, which is the same as [`DJANGO_SETTINGS_MODULE` env var](https://docs.djangoproject.com/en/stable/topics/settings/#designating-the-settings)
+- `strict_settings` which can be `true` or `false` (defaults to `true`), behavior is [specified here](https://github.com/typeddjango/django-stubs#how-to-use-a-custom-library-to-handle-django-settings)
```
Comment (adamchainz) [inline README.md]: Wordsmithing to use simpler sentences and more accurate terminology.

```suggestion
## Settings

django-stubs has a few settings, which you can list in:

* `pyproject.toml`, under the table `[tool.django-stubs]`
* `mypy.ini` under the table `[mypy.plugins.django-stubs]`

The supported settings are:

- `django_settings_module`, a string.

  Specify the import path of your settings module, the same as Django’s [`DJANGO_SETTINGS_MODULE` environment variable](https://docs.djangoproject.com/en/stable/topics/settings/#designating-the-settings).

- `strict_settings`, a boolean, default `true`.

  Set to `false` if using dynamic settings, as [described below](https://github.com/typeddjango/django-stubs#how-to-use-a-custom-library-to-handle-django-settings).
```

**Review by adamchainz — CHANGES_REQUESTED** [review]
Nice one.

File: README.md (hunk 3 above)
Comment (sobolevn) [reply README.md]: Much better, thank you!

File: README.md [hunk 4]
```diff
@@ -80,6 +80,23 @@ class MyModel(models.Model):
 
 * `django_stubs_ext.db.router.TypedDatabaseRouter` can be used as base when implementing custom database routers.
 
+## Settings
+
+django-stubs has a few settings, which you can list in:
+
+* `pyproject.toml`, under the table `[tool.django-stubs]`
+* `mypy.ini` under the table `[mypy.plugins.django-stubs]`
```
Comment (adamchainz) [inline README.md:88]: I made a comma inconsistency which will drive me nuts

```suggestion
* `pyproject.toml`, under the table `[tool.django-stubs]`
* `mypy.ini`, under the table `[mypy.plugins.django-stubs]`
```

**Review by adamchainz — APPROVED** [review]

**Pull request merged**